pkg/sgx/sgx.go

```diff
@@ -28,6 +44,12 @@ func AllDeviceDrivers() map[string]bool {
 
 func init() {
 	initOnce.Do(func() {
+		// Detecting mount points.
+		for mp := range allMountPoints {
+			if fi, err := os.Stat(mp); err == nil && !fi.IsDir() {
+				allMountPoints[mp] = true
+			}
+		}
```

Comment:
If user enables aesm socket attachment, we should ensure existence of aesm socket path inside init(), and then we only panic it if aesm socket is not found. Thus, we could remove `allMountPoints`.